catalogue/webapp/components/Work/Work.tsx

```diff
@@ -124,7 +124,7 @@ const Work: FunctionComponent<Props> = ({
       null,
     // We only send a langCode if it's unambiguous -- better to send no language
     // than the wrong one.
-    langCode: work.languages.length === 1 && work.languages[0].id,
+    langCode: work.languages.length === 1 && work.languages[0]?.id,
```

Comment:
why not optional chain the lot?

Reply:
I wondered about us enforcing the [prefer optional chaining](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-optional-chain.md) lint rule earlier. There's one for nullish coalescing too, which would probably make sense

Reply:
adding linting rule for this makes total sense